src/hardware_support/mod.rs

```diff
@@ -5,36 +5,53 @@ use os_release::OS_RELEASE;
 
 #[macro_use]
 mod macros;
+mod modules;
+
+use self::modules::Module;
 
 // NOTE: Distributions should provide their distro ID and associated packages here, if applicable.
 
 package!(amd_microcode {
-    "debian" => "amd64-microcode"
+    like "debian" => "amd64-microcode"
 });
 
 package!(intel_microcode {
-    "debian" => "intel-microcode"
+    like "debian" => "intel-microcode"
 });
 
 package!(system76_driver {
-    "debian" => "system76-driver"
+    like "debian" => "system76-driver"
 });
 
-pub fn append_packages(install_pkgs: &mut Vec<&'static str>) {
-    let distro = OS_RELEASE.id_like.as_str();
+package!(nvidia_driver {
+    like "debian", vendor "System76" => "system76-driver-nvidia",
+    like "debian" => "nvidia-driver-390"
```

Comment:
Since we have a separate NVIDIA ISO, we don't need to isntall nvidia-driver-390 from distinst